Src/album.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-, initial-scale=1.0">
+    <title>album</title>
+    <link rel="stylesheet" href="styles.css"/>
```

Comment:
Лучше использовать путь с использованием %PUBLIC_URL%
```suggestion
    <link rel="stylesheet" href="%PUBLIC_URL%/styles.css"/>
```

При сборке это заменится на корректный путь.

Это также относится и к картинкам